pylint/checkers/deprecated.py

```diff
@@ -154,3 +181,38 @@ def check_deprecated_method(self, node, inferred):
                 self.add_message(
                     "deprecated-argument", node=node, args=(arg_name, func_name)
                 )
+
+    def check_deprecated_class(self, node, mod_path=""):
+        """Checks if the class is deprecated"""
+
```

Comment:
I think we can reduce duplication by getting mod_name or mod_path and class_name before and then doing only one 
```python
if class_name in self.deprecated_classes(mod_name_or_path):
    self.add_message(
        "deprecated-class", node=node, args=(class_name, mod_name_or_path)
    )
 ```
 Like in https://github.com/PyCQA/pylint/pull/4419/files#diff-7d982c7ee8d920542a08ef8e7a73fddf8929599d2c13507a7b3b6bf2a4371b6fR157